packages/react-hooks/package.json

```diff
@@ -29,12 +29,12 @@
   },
   "devDependencies": {
     "@aries-framework/core": "^0.2.0",
-    "react": "^17.0.2",
+    "react": "^18.0.0",
```

Comment:
What if we make it a semver range 

E.g.:

```
>=17.0.0 <=19.0.0
```

or just

```
>=17.0.0
```

Then it doesn't have to be a breaking change

Reply:
I don't see any reason that would not work! You are a wise man Timo, this hadn't occurred to me 👏 I'll go ahead and update this shortly.